### PR TITLE
Added new features to Feature/details popup

### PR DESCRIPTION
**Pull request by melkamu372**
### Add new features to Feature/details-popup branch 

-  add popup functionality 
-  add javascript objects to store and access data
- JavaScript manipulate DOM elements
- JavaScript events to pop up for mobile and desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -129,6 +129,151 @@ label {
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 10;
+    background-color: #fff;
+    width: 80vw;
+    max-width: 90vw;
+    height: 90vh;
+    overflow-y: scroll;
+    overflow-x: hidden;
```
Comment (Ibizugbe) [inline style.css]: Nice job so far👍🏼. The functionalities of your popup works just fine😉. But let's try to make it match the given design both on mobile and on desktop😉.

Design
Screenshot 2023-08-02 at 08 18 25

Yours
Screenshot 2023-08-02 at 08 28 15
Screenshot 2023-08-02 at 08 28 33

Take note of the positioning of the elements, their sizes, font properties, and spacing.
Do well to also make it responsive as it is currently not responsive. ensure the popup on the mobile screen matches the given design😉.

File: script/popup.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+
+      <div class="project-card-container card-section-1">
+          <div class="projectone-header">
+            <h2>${data.title}</h2>
+          </div>
+          <div class="project-one-header-desktop">
+            <h2>${data.title}</h2>
+          </div>
+
```
Comment (Ibizugbe) [inline script/popup.js]: Nice job generating your project section dynamically😉. I suggest you kindly consider the spacings around the project section both on mobile and on desktop as your project is looking a bit different from the given design currently😉. see the image below😉.

yours
image


design
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @melkamu372 👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍🏼.
- [x] An array of object containing the project details has been created👍🏼.
- [x] Project section and modal popup is generated dynamically👍🏼.
- [x] Popup functionalities work as required👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @melkamu372, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- User clicks the button, the popup with details about the project appears ✔️
- User clicks the close (X) button, the popup disappears ✔️
- Modal popup data is populated with the data from the projects array ✔️
- Good use of Github flow ✔️


## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**